src/app/login/login.module.ts

```diff
@@ -4,12 +4,13 @@ import { FormsModule } from '@angular/forms';
 
 import { LoginComponent } from './login.component';
 
-
+import { MatInputModule } from '@angular/material/input';
 import { LoginRouterModule } from './login-router.module';
+import { MatButtonModule } from '@angular/material/button';
 
 @NgModule({
   imports: [
-    LoginRouterModule, FormsModule, CommonModule
+    LoginRouterModule, FormsModule, CommonModule, MatInputModule, MatButtonModule

```

Comment:
We should only be importing the `MaterialModule` from `shared/material.module`.  If the module you need is not in that file, please add it as an export in that file rather than adding it to an individual module.